app/controllers/verify/sessions_controller.rb

```diff
@@ -112,7 +112,7 @@ def idv_form
     end
 
     def initialize_idv_session
-      idv_session.params.merge!(profile_params)
+      idv_session.params = profile_params.to_h
```

Comment:
Yeah. It's the first step. It will be overwritten every time you submit the form with a full set of new values. After success you won't be able to come back to this point and overwrite succeeding changes. The params don't look like they have an initial value. Seems safe to me.